src/adaptors/beforeTests.js

```diff
@@ -39,6 +39,6 @@ module.exports = async function () {
   global.uniquePoolIdentifiersDB = new Set(
     (await axios.get('https://yields.llama.fi/distinctID')).data
       .filter((p) => p.project !== global.apy[0].project)
-      .map((p) => p.pool)
+      .map((p) => p.pool.toLowerCase())
```

Comment:
Happy to remove this changes

I added it because it was not catching all duplicate pools like this
![image](https://github.com/user-attachments/assets/9fad5bf9-edec-4bce-8054-36831eb93847)

-------------
This should now be fixed as I blacklisted uniswap from Merkl